js/components/post/post.js.jsx

```diff
@@ -1,7 +1,8 @@
 import { SpecialistPost, MajorPost, MinorPost } from "./post_components.js.jsx"
 
 import React from "react"
-import ReactDOM from "react-dom"
+import { createRoot } from "react-dom/client"
+import { createRef } from "react"
```

Comment:
Don't import this separately; you can do `React.createRef` below due to the previous react import